### ISSUE TITLE
The Expandable component needs a perceptible focus indicator

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#706]
## 🐛 Bug report

**Current Behavior**

As it stand currently, when receiving keyboard focus, the expandable has a very subtle CSS change that does not meet color contrast criteria:
<img width="847" alt="Screen Shot 2020-12-02 at 9 46 08 AM" src="https://user-images.githubusercontent.com/3613392/100887778-5b2d7d00-3483-11eb-86e3-2f8fbdcff0f6.png">

**Expected behavior**
Ideally, on focus this would render with an outline similar to other UI in the app that is clickable (e.g., `Button`).

**Reproducible example**
See https://design.sparkpost.com/components/expandable/

**Suggested solution(s)**
Incorporate one of the existing focus outline helpers for this component.
